library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -247,6 +253,9 @@ public DefaultBucket(long bucketId)
     public void setup(@NotNull ManagedStateContext managedStateContext)
     {
       this.managedStateContext = Preconditions.checkNotNull(managedStateContext, "managed state context");
+      if (!disableBloomFilter && bloomFilter == null) {
+        bloomFilter = new SliceBloomFilter(bloomFilterBitSize, 0.99);
+      }
```

Comment:
Yes, for ManagedStateImpl, bloom filter would be useless for a long run(But I still useful for certain period). ManagedTimeStateImpl should be useful for most of time. 
That's why I add the simple mechanism to check the efficiency of bloomfilter to dynamic unload it if it is useless.